backend/e2e-test/routes/v3/secret-reference.spec.ts

```diff
@@ -0,0 +1,153 @@
+import { createFolder, deleteFolder } from "e2e-test/testUtils/folders";
```

Comment:
We need tests to check secret references also work with both kinds of imports

Reply:
Alright

Reply:
Added